client/src/pages/reports/Show.js

```diff
@@ -43,6 +43,7 @@ import { toast } from "react-toastify"
 import { deserializeQueryParams } from "searchUtils"
 import utils from "utils"
 import { parseHtmlWithLinkTo } from "utils_links"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
 import AttendeesTable from "./AttendeesTable"
 import AuthorizationGroupTable from "./AuthorizationGroupTable"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).